guide/sections/part2/wis2-architecture.adoc

```diff
@@ -112,7 +113,14 @@ Refer to the _Manual on WIS_, Volume II for details.
 * A Global Discovery Catalogue amends discovery metadata records to add details of where one can subscribe to updates about the dataset at a Global Broker.
 * A Global Discovery Catalogue makes its content available for indexing by search engines.
 
-==== 2.4.5 Global Monitor
+==== 2.4.5 Global Replay
+* WIS2 may include a Global Replay.
+* A Global Replay enables Global Services and data consumers to search, query and subscribe to notification messages and monitoring events published by the Global Broker.
```

Comment:
The Global Replay provides Global Cache, Global Discovery Catalogue and data consumers with a mechanism to search, query and subscribe to **past** notification messages and monitoring events of interest as published by the Global Broker.